site/docs/expressions/scalar_functions.md

```diff
@@ -19,11 +19,14 @@ A function is a scalar function if that function takes in values from a single r
 ## Argument Types
 
 There are four main types of arguments: value arguments, type arguments, required enumerations, and optional enumerations.
+The first three types of arguments are considered positional arguments and must be specified in every invocation of the
```

Comment:
Doesn't feel quite right with new pattern.

Reply:
I reworded this entire section.  Please review.